model/src/main/resources/META-INF/openapi.yaml

```diff
@@ -107,8 +107,12 @@ components:
     iceberg:
       value:
         type: ICEBERG_TABLE
+        id: b874b5d5-f926-4eed-9be7-b2380d9810c0
```

Comment:
Should be enclosed by quotes as it is a String?

Reply:
quotes are optional for yaml in this case, it is automatically a string (same as `ICEBERG_TABLE` above)

Reply:
got it. previously uuid had quotes and below `metadataLocation` was having quotes. So, I got a doubt.